src/routes.php

```diff
@@ -1,38 +1,78 @@
 <?php
 $middlewares = \Config::get('lfm.middlewares');
+$prefix = \Config::get('lfm.prefix');
```

Comment:
@Rohos Would you please modified it to `$prefix = \Config::get('lfm.prefix', 'laravel-filemanager');` thus we can upgrade the package seamlessly .